docs/data/routes/docs/nextjs/deploying-to-production/export/en.md

```diff
@@ -66,11 +66,11 @@ Before exporting your application, you must:
 
    * `"next:export": "next export"`.
 
-   * For disconnected mode: `"export": "cross-env-shell JSS_MODE=disconnected PORT=3042 EXPORT_MODE=true \"npm-run-all --serial bootstrap next:build next:export\""`.
+   * When using content from Sitecore ("Connected Mode"): `"export:connected": "cross-env-shell EXPORT_MODE=true \"npm-run-all --serial bootstrap next:build next:export\""`.
 
-     Replace the `PORT` value with the port of your disconnected server.
+      Replace the `PORT` value with the port of your disconnected server.
```

Comment:
Looks like connected mode is now first. This comment about disconnected server port should be moved down after disconnected bullet.